src/make-parameterized-roles.js

```diff
@@ -42,6 +42,9 @@ module.exports.run = async function(options) {
       'project:<..>:*',
       'queue:get-artifact:project/<..>/*',
       'queue:route:index.project.<..>.*',
+      'queue:create-task:highest:project-<..>/*',
```

Comment:
Give all priority levels, otherwise they can't give `queue:create-task:low:projec..` to a client/role

Reply:
good point, thanks